src/disc-golf-events/AddEventComponent.tsx

```diff
@@ -20,11 +20,20 @@ const AddEventComponent: React.FC<AddEventProps> = ({ onClose, onEventAdded, set
         vacancies: ""
     });
 
+    const [touched, setTouched] = useState({
```

Comment:
In EditEventDialog there is useState<{ tournamentTitle?: boolean }> with specific type definition, while here there is no any type definition. Let's be consistent if we should use precise TypeScript types or not.
Also here default value is { tournamentTitle: false } while in EditEventDialog it is empty object {}. Let's be consistent here too.